### PR TITLE
Add note about viewing logs in Azure

### PR DESCRIPTION
**Pull request by lucyzhang929**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,10 @@ This extension collects usage data in order to help us improve your experience.
 
 For troubleshooting SQL Client issues, You can enable verbose logging by setting the `AzureFunctions_SqlBindings_VerboseLogging` app setting (in your `*.settings.json` file) to '1', 'true' or 'yes';
 
+### Logs
+
+Logs for function apps deployed in Azure can be viewed in the Function's monitoring tab. If logs are not showing in the monitoring tab, they can also be found in the Function App's [Log Stream]([url](https://learn.microsoft.com/en-us/azure/azure-functions/streaming-logs?tabs=azure-portal)) or in [Application Insights]([url](https://learn.microsoft.com/en-us/azure/azure-functions/analyze-telemetry-data#view-telemetry-in-application-insights)). More information on logging can be found [here](https://learn.microsoft.com/en-us/azure/azure-functions/functions-monitoring).
```
Comment (Charles-Gagnon) [inline README.md:86]:
```suggestion
Logs for function apps deployed in Azure can be viewed in the Function's monitoring tab. If logs are not showing in the monitoring tab, they can also be found in the Function App's [Log Stream]([url](https://learn.microsoft.com/azure/azure-functions/streaming-logs?tabs=azure-portal)) or in [Application Insights]([url](https://learn.microsoft.com/azure/azure-functions/analyze-telemetry-data#view-telemetry-in-application-insights)). More information on logging can be found [here](https://learn.microsoft.com/azure/azure-functions/functions-monitoring).
```

**Review by Charles-Gagnon — APPROVED** [review]

**Review by MaddyDev — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -81,6 +81,10 @@ This extension collects usage data in order to help us improve your experience.
 
 For troubleshooting SQL Client issues, You can enable verbose logging by setting the `AzureFunctions_SqlBindings_VerboseLogging` app setting (in your `*.settings.json` file) to '1', 'true' or 'yes';
 
+### Logs
+
+Logs for function apps deployed in Azure can be viewed in the Function's monitoring tab. If logs are not showing in the monitoring tab, they can also be found in the Function App's [Log Stream](https://learn.microsoft.com/azure/azure-functions/streaming-logs?tabs=azure-portal) or in [Application Insights](https://learn.microsoft.com/azure/azure-functions/analyze-telemetry-data#view-telemetry-in-application-insights). More information on logging can be found [here](https://learn.microsoft.com/azure/azure-functions/functions-monitoring).
```
Comment (chlafreniere) [inline README.md:86]: let's be consistent with casing here "Function Apps" everywhere would be great.

Comment (chlafreniere) [inline README.md:86]: I believe the tab is called "Monitor" on the portal

Comment (lucyzhang929) [reply README.md]: Updated to lower case to be consistent with MSDocs.

Comment (lucyzhang929) [reply README.md]: Yes Monitor is correct :)

**Pull request merged**